assets/src/block-templates/deep-dive/template.js

```diff
@@ -0,0 +1,54 @@
+import mainThemeUrl from '../main-theme-url';
+
+const {__} = wp.i18n;
+
+const topic = ['core/column', {}, [
+  ['core/group', {className: 'group-stretched-link'}, [
+    ['core/image', {
+      align: 'center',
+      className: 'force-no-lightbox force-no-caption is-style-rounded-180',
+      url: `${mainThemeUrl}/images/placeholders/placeholder-180x180.jpg`,
+      height: '180px',
```

Comment:
Needed to define `height` and `width` to avoid having differences between stored metadata

For instance 👇 
`style="object-fit:cover"/><figcaption`
VS
`style="object-fit:cover;width:180px;height:180px"/><figcaption`